app/controllers/admins_controller.rb

```diff
@@ -2,8 +2,8 @@ class AdminsController < ApplicationController
     before_action :authenticate_user!
     
     def index
-      @items=Item.all
-      @procedures=Procedure.all
+      @items=Item.where("status = 'pending'")
```

Comment:
- add spacing between the `=` sign
- use a hash too query `@items = Item.where(status: 'pending')`

Reply:
Fixed, also in several other files.